pkg/ovsdb/monitor.go

```diff
@@ -562,6 +564,37 @@ func unmarshalData(data []byte) (map[string]interface{}, error) {
 	return obj, nil
 }
 
+func (u *updater) isRowAppearOnWhere(data map[string]interface{}) (bool, error) {
+	if u.mcr.Where == nil {
+		return true, nil
+	}
+	var cond bool
+	for i := 0; i < len(*u.mcr.Where); i++ {
+		switch reflect.TypeOf((*u.mcr.Where)[i]).Kind() {
+		case reflect.Slice:
+			condition, ok := ((*u.mcr.Where)[i]).([]interface{})
+			if !ok {
+				return false, errors.New("unvalid condition")
```

Comment:
please add the condition's type. (I think it can be `array`, because we know that OVSDB condition is an array of 3 elements)

Reply:
Done